tmt.spec

```diff
@@ -90,6 +90,20 @@ Recommends:     qemu-system-x86-core
 
 %description -n tmt+provision-virtual %_metapackage_description
 
+%package -n     tmt+provision-bootc
+Summary:        Dependencies required for tmt bootc machine provisioner
+Provides:       tmt-provision-bootc == %{version}-%{release}
+%if 0%{?fedora} < 40
+Obsoletes:      tmt-provision-bootc < %{version}-%{release}
+%endif
```

Comment:
There was no `tmt-provision-bootc` package before, so I think we can drop the `Obsoletes` here.